marklogic-data-hub-central/ui/src/components/column-selector/column-selector.test.tsx

```diff
@@ -50,6 +50,12 @@ describe('Column selector component', () => {
     expect(applyButton.onclick).toHaveBeenCalledTimes(1);
   });
 
+  test('Verify apply button is disabled when no properties selected', () => {
+    const { getByText } = render(<ColumnSelector {...{...defaultProps, selectedPropertyDefinitions: [] }} />);
+    const applyButton = getByText('Apply');
+    expect(applyButton).toBeDisabled();
```

Comment:
No need to change anything, just an FYI that you can use rerender to render a component with desired props in an existing test, if its a one line assertion.